code/modules/research/designs/computer_part_designs.dm

```diff
@@ -0,0 +1,251 @@
+////////////////////////////////////////
+///////////Computer Parts///////////////
+////////////////////////////////////////
+
+/datum/design/disk/normal
+	name = "hard disk drive"
+	id = "hdd_basic"
+	req_tech = list("programming" = 1, "engineering" = 1)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 400, MAT_GLASS = 100)
+	build_path = /obj/item/weapon/computer_hardware/hard_drive
+	category = list("Computer Parts")
+
+/datum/design/disk/advanced
+	name = "advanced hard disk drive"
+	id = "hdd_advanced"
+	req_tech = list("programming" = 2, "engineering" = 2)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 800, MAT_GLASS = 200)
+	build_path = /obj/item/weapon/computer_hardware/hard_drive/advanced
+	category = list("Computer Parts")
+
+/datum/design/disk/super
+	name = "super hard disk drive"
+	id = "hdd_super"
+	req_tech = list("programming" = 3, "engineering" = 3)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 1600, MAT_GLASS = 400)
+	build_path = /obj/item/weapon/computer_hardware/hard_drive/super
+	category = list("Computer Parts")
+
+/datum/design/disk/cluster
+	name = "cluster hard disk drive"
+	id = "hdd_cluster"
+	req_tech = list("programming" = 4, "engineering" = 4)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 3200, MAT_GLASS = 800)
+	build_path = /obj/item/weapon/computer_hardware/hard_drive/cluster
+	category = list("Computer Parts")
+
+/datum/design/disk/small
+	name = "solid state drive"
+	id = "ssd_small"
+	req_tech = list("programming" = 2, "engineering" = 2)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 800, MAT_GLASS = 200)
+	build_path = /obj/item/weapon/computer_hardware/hard_drive/small
+	category = list("Computer Parts")
+
+/datum/design/disk/micro
+	name = "micro solid state drive"
+	id = "ssd_micro"
+	req_tech = list("programming" = 1, "engineering" = 1)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 400, MAT_GLASS = 100)
+	build_path = /obj/item/weapon/computer_hardware/hard_drive/micro
+	category = list("Computer Parts")
+
+
+// Network cards
+/datum/design/netcard/basic
+	name = "network card"
+	id = "netcard_basic"
+	req_tech = list("programming" = 2, "engineering" = 1)
+	build_type = IMPRINTER
+	materials = list(MAT_METAL = 250, MAT_GLASS = 100, "sacid" = 20)
+	build_path = /obj/item/weapon/computer_hardware/network_card
+	category = list("Computer Parts")
+
+/datum/design/netcard/advanced
+	name = "advanced network card"
+	id = "netcard_advanced"
+	req_tech = list("programming" = 4, "engineering" = 2)
+	build_type = IMPRINTER
+	materials = list(MAT_METAL = 500, MAT_GLASS = 200, "sacid" = 20)
+	build_path = /obj/item/weapon/computer_hardware/network_card/advanced
+	category = list("Computer Parts")
+
+/datum/design/netcard/wired
+	name = "wired network card"
+	id = "netcard_wired"
+	req_tech = list("programming" = 5, "engineering" = 3)
+	build_type = IMPRINTER
+	materials = list(MAT_METAL = 2500, MAT_GLASS = 400, "sacid" = 20)
```

Comment:
These cards require metal, but the imprinter is at the moment incapable of containing metal.